public/index.html

```diff
@@ -71,10 +71,10 @@ <h5 class="card-title">Development</h5>
                                 <div class="col-md-6" id="totalDevelopmentTask"></div>
                             </div>
                         </div>
-                    </div>
+                    </span>
                 </div>
                 <div class="col-md-4 col-12">
-                    <div class="card text-white mb-3" style="max-width: 18rem; background-color:#00bfa5 ;">
+                    <span id="Marketing" type="button" class="card text-white mb-3" style="max-width: 18rem; background-color:#00bfa5 ;">
```

Comment:
Please fix the span back to div this might create some problems with small screen, and type button is not needed it will work without that.

Reply:
k